packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+import { ApiReferencesTable, MEDIUM_GAP_SIZE, LARGE_GAP_SIZE } from './ApiReferencesTable';
+import { IApiProperty, IApiInterfaceProperty, IApiEnumProperty, IMethod, IApiReferencesTableSetProps } from './ApiReferencesTableSet.types';
+import { IEnumTableRowJson, ITableRowJson, ITableJson } from 'office-ui-fabric-react/lib/common/DocPage.types';
+import { PropertyType } from '../../utilities/parser/index';
+import { extractAnchorLink } from '../../utilities/extractAnchorLink';
+
+export interface IApiReferencesTableSetState {
+  properties: Array<IApiProperty>;
+  showSeeMore: boolean;
+}
+
+const apiReferencesTableTopMargin = mergeStyles({
+  marginTop: '40px'
+});
+
+const TITLE_LINE_HEIGHT = 31.5;
+
+export class ApiReferencesTableSet extends React.Component<IApiReferencesTableSetProps, IApiReferencesTableSetState> {
+  constructor(props: IApiReferencesTableSetProps) {
+    super(props);
+
+    this.state = {
+      properties: this._generatePropertyArray(),
+      showSeeMore: false
+    };
+  }
+
+  public render(): JSX.Element {
+    const { className } = this.props;
+    return (
+      <Stack gap={LARGE_GAP_SIZE} className={css(apiReferencesTableTopMargin, className)}>
+        {this._renderFirst()}
+        {this._renderEach()}
+      </Stack>
+    );
+  }
+
+  public componentDidMount(): void {
+    window.addEventListener('hashchange', this._onHashChange);
+  }
+
+  public componentWillUnmount(): void {
+    window.removeEventListener('hashchange', this._onHashChange);
+  }
+
+  public componentDidUpdate(prevProps: IApiReferencesTableSetProps, prevState: IApiReferencesTableSetState): void {
+    if (prevState.showSeeMore === false && this.state.showSeeMore === true) {
+      const hash = extractAnchorLink(window.location.hash);
+      const el = document.getElementById(hash);
+
+      if (el) {
+        // update scroll position
+        window.scrollTo({ top: el.offsetTop - TITLE_LINE_HEIGHT /* title */, behavior: 'smooth' });
```

Comment:
Shame you need the offset here otherwise [Element.scrollIntoView](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView) would have worked.

Reply:
You might consider sharing this function tho

Reply:
I'm planning on improving the scrolling in a next iteration - I'll keep this in mind :)